vagrant/bootstrap.sh

```diff
@@ -7,6 +7,9 @@ sudo apt-get update
 # Upgrade all currently installed packages
 sudo apt-get -y upgrade
 
+# Install attr package on the machine
+sudo apt-get install -y attr
+
 # Python3 versions
 sudo apt-get install -y python3 python3-pip python3-venv
```

Comment:
Wouldn't it be better to just add attr to this list? You can edit the comment to say `Install system dependencies`

Reply:
Sure, it makes sense. I will take care of this and update it.